crates/telio-model/src/features.rs

```diff
@@ -251,6 +248,26 @@ pub struct FeaturePaths {
     pub force: Option<PathType>,
 }
 
+impl FeaturePaths {
```

Comment:
This was moved from later in the file with no other changes to this block of code